src/modules/flight_mode_manager/tasks/AutoFollowTarget/FlightTaskAutoFollowTarget.cpp

```diff
@@ -187,22 +189,16 @@ Vector3f FlightTaskAutoFollowTarget::calculate_drone_desired_position(Vector3f t
 	return drone_desired_position;
 }
 
-Vector3f FlightTaskAutoFollowTarget::calculate_target_position_filtered(Vector3f pos_ned_est, Vector3f vel_ned_est,
-		Vector3f acc_ned_est)
+Vector3f FlightTaskAutoFollowTarget::calculate_target_position_filtered(Vector3f pos_ned_est, Vector3f vel_ned_est)
 {
 	// Reset the smoothness filter once the target position estimate is available
 	if (!PX4_ISFINITE(_target_position_filter.getState()(0)) || !PX4_ISFINITE(_target_position_filter.getState()(1))
 	    || !PX4_ISFINITE(_target_position_filter.getState()(2))) {
 		_target_position_filter.reset(pos_ned_est);
```

Comment:
Filter will converge faster if the `vel_ned_est` is also reset here.

Also - we may want to add the velocity components of the filter to this check .. in case any of them became !finite.